p2p/tests/identify.rs

```diff
@@ -104,6 +104,7 @@ async fn rust_node_to_rust_node() -> anyhow::Result<()> {
 }
 
 #[tokio::test]
+#[ignore = "TODO: Add override for reducer"]
```

Comment:
What is this TODO about?

Reply:
This test and same named test in `p2p/tests/kademlia.rs` work by having override effects so that node can send invalid data, I need to implement this functionality for reducer.